tests/shell/shell_recorder/tutorial_wrapper/CMakeLists.txt

```diff
@@ -14,8 +14,10 @@ add_s_test (conditionals "${CMAKE_SOURCE_DIR}/src/plugins/conditionals/README.md
 add_s_test (hosts "${CMAKE_SOURCE_DIR}/src/plugins/hosts/README.md")
 add_s_test (line "${CMAKE_SOURCE_DIR}/src/plugins/line/README.md")
 add_s_test (mathcheck "${CMAKE_SOURCE_DIR}/src/plugins/mathcheck/README.md")
-add_s_test (mozpref "${CMAKE_SOURCE_DIR}/src/plugins/mozpref/README.md")
+add_s_test (mozpref "${CMAKE_SOURCE_DIR}/src/plugins/mozprefs/README.md")
 add_s_test (tutorial_cascading "${CMAKE_SOURCE_DIR}/doc/tutorials/cascading.md")
-add_s_test (tutorial_mount "${CMAKE_SOURCE_DIR}/doc/tutorials/mount.md")
 add_s_test (kdb-complete "${CMAKE_SOURCE_DIR}/doc/help/kdb-complete.md")
 add_s_test (xerces "${CMAKE_SOURCE_DIR}/src/plugins/xerces/README.md")
+
+# Only works with super user privileges, since it writes to `/etc/hosts`:
+# add_s_test (tutorial_mount "${CMAKE_SOURCE_DIR}/doc/tutorials/mount.md")
```

Comment:
[Yes](https://github.com/ElektraInitiative/libelektra/issues/1139), that is [correct](https://github.com/ElektraInitiative/libelektra/issues/944). For now only the stripping part works as far as I can tell.